pubsublite: use of 10GB int consts breaks 32 bit systems

```
697s # cloud.google.com/go/pubsublite/pscompat
697s src/cloud.google.com/go/pubsublite/pscompat/settings.go:125:21: cannot use 1e10 (untyped float constant 1e+10) as int value in struct literal (truncated)
```

and 

```
# cloud.google.com/go/pubsublite/internal/wire [cloud.google.com/go/pubsublite/internal/wire.test]
src/cloud.google.com/go/pubsublite/internal/wire/settings_test.go:41:34: cannot use 1e10 (untyped float constant 1e+10) as int value in assignment (truncated)
src/cloud.google.com/go/pubsublite/internal/wire/settings_test.go:136:36: cannot use 1e10 (untyped float constant 1e+10) as int value in assignment (truncated)
```

--- a/pubsublite/internal/wire/settings_test.go
+++ b/pubsublite/internal/wire/settings_test.go
@@ -38,7 +38,7 @@ func TestValidatePublishSettings(t *testing.T) {
 				// These have no max bounds, check large values.
 				settings.DelayThreshold = 24 * time.Hour
 				settings.Timeout = 24 * time.Hour
-				settings.BufferedByteLimit = 1e10
+				settings.BufferedByteLimit = 1e9
 			},
 			wantErr: false,
 		},
@@ -133,7 +133,7 @@ func TestValidateReceiveSettings(t *testing.T) {
 				settings.Partitions = []int{5, 3, 9, 1, 4, 0}
 				// These have no max bounds, check large values.
 				settings.MaxOutstandingMessages = 100000
-				settings.MaxOutstandingBytes = 1e10
+				settings.MaxOutstandingBytes = 1e9
 				settings.Timeout = 24 * time.Hour
 			},
 			wantErr: false,

--- a/pubsublite/pscompat/settings.go
+++ b/pubsublite/pscompat/settings.go
@@ -122,7 +122,7 @@ var DefaultPublishSettings = PublishSettings{
 	CountThreshold:    100,
 	ByteThreshold:     1e6,
 	Timeout:           7 * 24 * time.Hour,
-	BufferedByteLimit: 1e10,
+	BufferedByteLimit: 1e9,
 	EnableIdempotence: true,
 }
 

